examples/sprite_camera_follow/main.rs

```diff
@@ -0,0 +1,184 @@
+
+        for (_, transform) in (&players, &mut transforms).join() {
+            transform.translation.x += x_move as f32 * 5.0;
+            transform.translation.y += y_move as f32 * 5.0;
+        }
+    }
+}
+
+fn load_sprite_sheet(world: &mut World, png_path: &str, ron_path: &str) -> SpriteSheetHandle {
+    let texture_handle = {
+        let loader = world.read_resource::<Loader>();
+        let texture_storage = world.read_resource::<AssetStorage<Texture>>();
+        loader.load(
+            png_path,
+            PngFormat,
+            TextureMetadata::srgb_scale(),
+            (),
+            &texture_storage,
+        )
+    };
+    let mut material_texture_set = world.write_resource::<MaterialTextureSet>();
+    let texture_id = material_texture_set.len() as u64;
+    material_texture_set.insert(texture_id, texture_handle);
+
+    let loader = world.read_resource::<Loader>();
+    let sprite_sheet_store = world.read_resource::<AssetStorage<SpriteSheet>>();
+    loader.load(
+        ron_path,
+        SpriteSheetFormat,
+        texture_id,
+        (),
+        &sprite_sheet_store,
+    )
+}
+
+// Initialize a background
+fn init_background_sprite(world: &mut World, sprite_sheet: &SpriteSheetHandle) -> Entity {
+    let mut transform = Transform::default();
+    transform.translation.z = -10.0;
+    let sprite = SpriteRender {
+        sprite_sheet: sprite_sheet.clone(),
+        sprite_number: 0,
+        flip_horizontal: false,
+        flip_vertical: false,
+    };
+    world.create_entity().with(transform).with(sprite).build()
+}
+
+// Initialize a sprite as a reference point at a fixed location
+fn init_reference_sprite(world: &mut World, sprite_sheet: &SpriteSheetHandle) -> Entity {
+    let mut transform = Transform::default();
+    transform.translation.x = 100.0;
+    transform.translation.y = 0.0;
+    let sprite = SpriteRender {
+        sprite_sheet: sprite_sheet.clone(),
+        sprite_number: 0,
+        flip_horizontal: false,
+        flip_vertical: false,
+    };
+    world
+        .create_entity()
+        .with(transform)
+        .with(sprite)
+        .with(Transparent)
+        .build()
+}
+
+fn init_player(world: &mut World, sprite_sheet: &SpriteSheetHandle) -> Entity {
+    let mut transform = Transform::default();
+    transform.translation.x = 0.0;
+    transform.translation.y = 0.0;
+    let sprite = SpriteRender {
+        sprite_sheet: sprite_sheet.clone(),
+        sprite_number: 1,
+        flip_horizontal: false,
+        flip_vertical: false,
+    };
+    world
+        .create_entity()
+        .with(transform)
+        .with(Player)
+        .with(sprite)
+        .with(Transparent)
+        .build()
+}
+
+fn init_camera(world: &mut World, parent: Entity) {
+    let mut transform = Transform::default();
+    transform.translation.z = 1.0;
+    world
+        .create_entity()
+        .with(Camera::from(Projection::orthographic(
+            -250.0, 250.0, 250.0, -250.0,
```

Comment:
If this gets merged after the nalgebra pr (depending on how fast you are to fix my review, you might be before), you'll need to change the order of some of those values.